src/api/Webpack/index.ts

```diff
@@ -10,4 +9,12 @@ class DummyWebpackApi implements IBaseWebpackApi {
     }
 }
 
+export interface ModImplementation {
```

Comment:
`ModImplementation` should not be bound to `Webpack` module as it's meant to be a global definition for client mod implementations

Reply:
Yeah, I was moving it rn